src/njoy21/input/DTFR/Card4/HollerithNames.hpp

```diff
@@ -0,0 +1,18 @@
+struct HollerithNames {
+  using Value_t = std::vector<std::string >;
+  static std::string name(){ return "hollerithNames"; }
+
+  static std::string description() {
+    return 
+      "HollerithNames is a vector of the six-character hollerith names for\n"
+      "edits. User may enter names for as many cards as needed. Ther will be\n"
+      "(iptotl - 3) names read. Each name is delimited with a '.";
```

Comment:
I would say (to be explicit):

`Each name is delimited with a ' (apostrophe).`

It looks a bit confusing as it reads right now—even though that is what is in the manual.